src/editor/Editor.js

```diff
@@ -451,6 +452,9 @@ define(function (require, exports, module) {
                 self.document._toggleMasterEditor(self);
             }
         });
+        this.on("providerAvailable", function (event, editor) {
```

Comment:
This is odd.  I think you're just doing it for debugging.  Just move the `console.log` up to where you trigger the event, instead.